cli/ballerina-cli/src/main/java/io/ballerina/cli/task/RunTestsTask.java

```diff
@@ -199,6 +199,9 @@ public void execute(Project project) {
             if (isSingleTestExecution || isRerunTestExecution) {
                 suite.setTests(TesterinaUtils.getSingleExecutionTests(suite, singleExecTests));
             }
+            if (project.kind() == ProjectKind.SINGLE_FILE_PROJECT) {
```

Comment:
```suggestion
            if (project.kind() == ProjectKind.SINGLE_FILE_PROJECT && project.sourceRoot() != null) {
```

Reply:
Ive tried this. However it still doesnt seem to stop the spotbugs from throwing warnings